multimodal/tests/unittests/test_text_detection.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import pytest
+import requests
+from PIL import Image
+
+from autogluon.multimodal import MultiModalPredictor
+
+
+def download_sample_images():
+    url = "https://raw.githubusercontent.com/open-mmlab/mmocr/main/demo/demo_text_det.jpg"
+    image = Image.open(requests.get(url, stream=True).raw)
+    mmocr_image_name = "demo.jpg"
+    image.save(mmocr_image_name)
+
+    return mmocr_image_name
+
+
+@pytest.mark.parametrize(
+    "checkpoint_name",
+    ["textsnake_r50_fpn_unet_1200e_ctw1500", "dbnet_r18_fpnc_1200e_icdar2015"],
+)
+def test_mmocr_text_detection_inference(checkpoint_name):
+    mmocr_image_name = download_sample_images()
+
+    predictor = MultiModalPredictor(
+        hyperparameters={
+            "model.mmocr_text_detection.checkpoint_name": checkpoint_name,
+        },
+        pipeline="ocr_text_detection",
+    )
+
+    predictor.predict({"image": [mmocr_image_name]})
```

Comment:
Please add some tests here to check if the results are good. For example, what is the desired output dimension? Otherwise, we don't know if the forward is done correctly.

Reply:
OK.

Reply:
Let's just test one model then. In this case, one fixed input image, one model, the output should be certain. We can add more test cases later. Thank you.

Reply:
OK.